package.json

```diff
@@ -2,6 +2,8 @@
   "name": "@netlify/build-info",
   "version": "3.0.1",
   "description": "Build info utility",
+  "type": "module",
+  "exports": "./src/main.js",
   "main": "./src/main.js",
```

Comment:
We need to keep `main` because some libraries (including our own `@netlify/zip-it-and-ship-it`) have been known to load modules' `package.json` and use their `main` property.